brands/amenity/cafe.json

```diff
@@ -973,6 +994,17 @@
       "official_name": "Starbucks Coffee",
       "takeaway": "yes"
     }
+  },
+    "amenity/cafe|Tealife": {
+    "tags": {
+      "amenity": "cafe",
+      "brand": "Tealife",
+      "brand:wikidata": "Q95722097",
```

Comment:
Just a quick review of https://www.wikidata.org/wiki/Q95722097 shows a few issues.
* It's inconsistent whether the brand is "Tealife" or "Tealive".
* Missing "instance of" property. This one is really important and should be "business" or some subclass of it.
* Bizarre "enemy of" property.  There are warnings that this kind of property is only appropriate for fictional characters.

Reply:
Firstly, is Tealive. That is my fault as I cluttered it. https://www.tealive.com.my/
. I had corrected wikidata per your feedback